tools/runqlat.bt

```diff
@@ -22,6 +22,10 @@
 #define TASK_RUNNING 0
 #endif
 
+config = {
+	log_warnings=0;
```

Comment:
Document what warnings are suppressed? 

Same for below

Reply:
Aside: in rust there is `#![allow(..)]` attribute syntax to suppress warnings/errors on specific parts of the source. This might compose well with our probe attributes desire, as attributes could be attached to arbitrary points of the AST. Suppression could be propagated down using a visitor and then diganostic class could check for suppression. It'd work best if we had warning categories just like rust, so the suppression is more targetted.

Reply:
Yeah, I need to give our whole warning thing more thought. There is quite a mix (as @amscanne mentioned above) of runtime, compile time, and even developer-targeted warnings.